frontend/package-lock.json

```diff
@@ -1263,8 +1263,6 @@
     },
     "node_modules/@kie/lock-treatment-tool": {
       "version": "0.1.0",
-      "resolved": "https://registry.npmjs.org/@kie/lock-treatment-tool/-/lock-treatment-tool-0.1.0.tgz",
-      "integrity": "sha512-KkUPkdWdjfp8T9EmNybGHAbvXdxdOcotgG21Lyk92e4ASFb/L0x5nJMOCUdX2ras5hD39TIqT4R2v8Z56UzDxA==",
```

Comment:
Probably this can be omitted from the PR.

Reply:
Thanks for noticing!